modules/govuk_jenkins/manifests/packages/tfenv.pp

```diff
@@ -0,0 +1,26 @@
+# == Class: govuk_jenkins::packages::tfenv
+#
+# Installs tfenv https://github.com/tfutils/tfenv
+# to manage multiple versions of terraform installed concurrently on the system
+#
+# === Parameters
+#
+# [*terraform_versions*]
+#   The list of terraform versions to be installed
+#
+#
+class govuk_jenkins::packages::tfenv (
+  $terraform_versions = ['0.11.14', '0.13.6'],
+){
+
+  class { '::tfenv':
+    manage_user    => false,
+    manage_group   => false,
+    tfenv_user     => 'jenkins',
+    tfenv_group    => 'jenkins',
```

Comment:
Feels like we could just hardcode these in the class. It's unlikely we're going to want to run tfenv as anything other than the jenkins:jenkins. And we could remove the manage_user / manage_group bits, since they're always false in practice.

Reply:
Yep, can fix that, we copied across the code contents of the puppet module: https://forge.puppet.com/modules/sergk/tfenv because of some dependencies conflict